app/resources/base.html.template

```diff
@@ -210,11 +199,7 @@
             >{% trans "Terms of Service" %}</a>
       {% endblock sidenav %}
     </div>
-    {% endif %}
-    {% if not env.amp %}
     <div id="overlay" class="overlay" onclick="hideSideMenu()"></div>
```

Comment:
Don't you need to remove this element in AMP?

Reply:
Ping.

Reply:
There is no way to show this div on the AMP page as same as navigator menu and onclick doesn't allow to use in AMP page.

Reply:
Oops sorry I didn't notice that it's still inside ```{% if not env.amp %}```. Then it should be fine.